lib/_tools.scss

```diff
@@ -76,14 +75,10 @@
         -webkit-flex-flow: row wrap;
         -ms-flex-flow: row wrap;
         flex-flow: row wrap;
-        -webkit-flex: 1 1 auto; // [2]
-        -ms-flex: 1 1 auto; // [2]
-        flex: 1 1 auto; // [2]
-
-        margin-right: 0; // [3]
+        margin-right: 0; // [2]
```

Comment:
There is an `rtl` button the demo so you can toggle it on and off to see it working: http://bbc.github.io/gel-grid/. The display order of the items is reversed meaning the margin doesn't need to be changed.